src/index.js

```diff
@@ -26,47 +24,13 @@ const PLATFORM_MAPPING = {
   freebsd: "freebsd",
 };
 
-function getInstallationPath(callback) {
-  // `npm bin` will output the path where binary files should be installed
-  exec("npm bin", function (err, stdout, stderr) {
```

Comment:
As stated in the description, this is only npm compatible, it was not compatible with Yarn and pnmp (same thing for `npm_config_prefix` env var).